mlos_bench/mlos_bench/tests/services/remote/mock/mock_auth_service.py

```diff
@@ -26,10 +26,7 @@ def __init__(self, config: Optional[Dict[str, Any]] = None,
                  methods: Union[Dict[str, Callable], List[Callable], None] = None):
         super().__init__(
             config, global_config, parent,
-            self.merge_methods(methods, [
-                self.get_access_token,
-                self.get_auth_headers,
-            ])
+            self.merge_methods(methods, [self.get_access_token, self.get_auth_headers])
```

Comment:
Kinda prefer the one per line version.  It's more consistent across many of the other services.

Reply:
mee too, but pylint complains about code duplication (here and in auth_service proper)

Reply:
Maybe disable that warning explicitly for that site then?